### PR TITLE
Rhdevdocs 2752

### LINKED ISSUE #29909: [enterprise-4.7] A sample CustomResource file has a syntax error in Installing OpenShift Logging
### Which section(s) is the issue in?

OCP 4.7 (latest as of 28-Feb-2021)

A sample yaml file of OpenShift Logging instance

https://docs.openshift.com/container-platform/4.7/logging/cluster-logging-deploying.html#cluster-logging-deploy-cli_cluster-logging-deploying

### What needs fixing?

The part of the sample file seems to be wrong in syntax.

```
spec:
...
  logStore:
    type: "elasticsearch"  
...
    elasticsearch:
...
     proxy: 
        limits:
          memory: 256Mi
        requests:
           memory: 256Mi
```

It should be the following, because when I apply the sample file, by "$oc create -f [the sample file]", the command returned the error.

```
spec:
...
  logStore:
    type: "elasticsearch"  
...
    elasticsearch:
...
     proxy:
        resources: 
            limits:
              memory: 256Mi
            requests:
               memory: 256Mi
```

```
$ oc create -f [the sample CustomResrouce yaml file]
The ClusterLogging "instance" is invalid: spec.logStore.elasticSearch.proxy.resources: Required Vaule
```

It's just a small mistake but just for sure.
(I googled and search in the issue list but not found anything on this topic.)

### LINKED ISSUE #30052: [enterprise-4.7] Indentation problem on logging installation docs
### Which section(s) is the issue in?
https://docs.openshift.com/container-platform/4.7/logging/cluster-logging-deploying.html

Hey team,
If you search `memory: "8Gi"` on the page, you'll see that there are 2 results where one of them's resource section's indentation had slid one tab. (Number 7 on yml)

### What needs fixing?
The resource tab needs to be fixed otherwise the operator doesn't reconcile the requested memory by CU.

### PR DESCRIPTION
**Pull request by rolfedh**
[RHDEVDOCS-2752](https://issues.redhat.com/browse/RHDEVDOCS-2752) Fix issue: [enterprise-4.7] Indentation problem on logging installation docs 
Fixes #30052, same as https://issues.redhat.com/browse/RHDEVDOCS-2752
Also fixes [enterprise-4.7] A sample CustomResource file has a syntax error in Installing OpenShift Logging #29909
Also fixes https://bugzilla.redhat.com/show_bug.cgi?id=1933110
[enterprise-4.5]
[enterprise-4.6]
[enterprise-4.7]
[enterprise-4.8]

### CODE REVIEW

**Comment by rolfedh** [comment]
@anpingli  Please verify the changes to the yaml in this PR. Thank you.

File: modules/cluster-logging-about-crd.adoc [hunk 1]
```diff
@@ -43,10 +43,11 @@ spec:
         request:
           memory: 8G
       proxy:
-        limits:
-          memory: 100Mi
-        requests:
-          memory: 100Mi
+        resources:
```
Comment (anpingli) [inline modules/cluster-logging-about-crd.adoc:46]: Any reason using 100Mi? How about use the default values?
          limits:
            memory: 256Mi
          requests:
            memory: 256Mi

**Review by anpingli — COMMENTED** [review]
Shall we use the default memory 256Mi? The other part /lgtm

**Comment by rolfedh** [comment]
> Shall we use the default memory 256Mi? The other part /lgtm

Updated to 256Mi. Thanks.

**Review by codyhoag — APPROVED** [review]
LGTM

**Pull request merged**